libtsuba/src/RDGPartHeader.cpp

```diff
@@ -33,6 +33,9 @@ const char* kEdgeEntityTypeIDDictionaryKey =
 const char* kNodeEntityTypeIDNameKey = "kg.v1.node_entity_type_id_name";
 // Name maps from Atomic Edge Entity Type ID to set of string names for the Edge Entity Type ID
 const char* kEdgeEntityTypeIDNameKey = "kg.v1.edge_entity_type_id_name";
+// List of node and edge indexed columns
+const char* kNodePropertyIndexColumnsKey = "kg.v1.node_property_index_columns";
+const char* kEdgePropertyIndexColumnsKey = "kg.v1.edge_property_index_columns";
```

Comment:
Hey @tylershunt, given that these key names seem to have no bearing on the RDG version, is it wise to keep calling them v1?

Reply:
These are versions of the key namespace. I don't see a reason to change them at this time, we're just adding keys.

Reply:
Everything is an IDspace to you these days.  It makes me feel a bit better about it though.